jointwt/twtxt
# Add domain on profile and timeline

##### Summary
This will add the domain to the profile and timeline feed.

##### Screenshots
![image](https://user-images.githubusercontent.com/5929807/101904923-59418780-3c02-11eb-934a-b4d6cab99629.png)
![image](https://user-images.githubusercontent.com/5929807/101904983-6a8a9400-3c02-11eb-8b15-b90ad8802a57.png)

##### Test Plan
- Make sure you have a post on your timeline from someone on the same pod
- Make sure you have a post on your timeline from someone on another pod

- The post on your timeline should not have a domain
- The post on the other timeline should have a domain

- The profile of the user on your pod should not have a domain
- The profile of the user on the other pod should  have a domain

##### Additional Information
This is **not ready**

- [ ]  Change the interpolation to be only the domain (it's currently the full url)
- [ ]  The domain shows for both internal and external profiles

First attempt to solve https://github.com/jointwt/twtxt/issues/284
